bin/test/TestScripts.scala

```diff
@@ -57,7 +57,7 @@ class TestScripts {
       s"bin/scalac script did not run properly. Output:$lineSep$dotcOutput"
     )
 
-    val (retDotr, dotrOutput) = executeScript("./bin/scala HelloWorld")
+    val (retDotr, dotrOutput) = executeScript("./bin/scala -M HelloWorld")
```

Comment:
this is slightly confusing, since just passing `-M` to Scala CLI, without any inputs or `-cp` will just fail.
I know it doesn't fail because `-cp` is passed elsewhere, but this line looks like it should fail, technically.